src/frontend/components/SøknadsSteg/Oppsummering/OppsummeringSteg/Eøs/EøsBarnOppsummering.tsx

```diff
@@ -50,6 +50,21 @@ const EøsBarnOppsummering: React.FC<Props> = ({ settFeilAnchors, nummer, barn }
                             toSlektsforholdSpråkId(barn.søkersSlektsforhold.svar),
                     })}
                 />
+                {barn.søkersSlektsforholdSpesifisering.svar && (
+                    <OppsummeringFelt
+                        tittel={
+                            <SpråkTekst
+                                id={
```

Comment:
kanskje man kunne dratt ut dette? er nesten lik som tittelen over

Reply:
` const tittelSpmOppsummering = (søknadSpørsmål: ISøknadSpørsmål<Slektsforhold | string>) => (
        <SpråkTekst
            id={eøsBarnSpørsmålSpråkId[søknadSpørsmål.id]}
            values={{ barn: barnetsNavnValue(barn, intl) }}
        />`
        
        `  tittel={tittelSpmOppsummering(
                        eøsSøkerSpørsmålSpråkId[EøsBarnSpørsmålId.søkersSlektsforhold]
                    )}`
                    
                    
Prøvde å lage noe slik, men fikk det ikke så mye kortere 🤔

Reply:
Hmm, tenkte feks:

`const tittelSpm = (spørsmålId: EøsBarnSpørsmålId) => (<SpråkTekst
                            id={eøsBarnSpørsmålSpråkId[spørsmålId]}
                            values={{ barn: barnetsNavnValue(barn, intl) }}
                        />)`
                        
  `tittel={tittelSpmOppsummering(barn.søkersSlektsforholdSpesifisering.id)}`

så blir det en linje i stedet for 9

Reply:
Så kan den gjenbrukes og på det forrige spørsmålet i samme fil på en måte